vent/io/__init__.py

```diff
@@ -1 +1,78 @@
-
+from typing import Any, Tuple
+
+import pigpio
+import configparser
+from importlib import import_module
+from .devices.sensors import AnalogSensor, SFM3200
+from .devices.valves import OnOffValve, PWMControlValve
+
+
+class Ventilator:

```

Comment:
probs a bad idea to have a Ventilator object in this project unless i guess if it's the main object

Reply:
I swear the hardest part of writing this piece of code is coming up with a name for that thing